contrib/buildahimage/centos7/Dockerfile

```diff
@@ -8,19 +8,32 @@ FROM centos:7
 
```

Comment:
Oh, just realized this is `centos7`.  My #4019 simply clobbers this whole directory, I don't think it's needed/relevant anymore (but please correct me if I'm wrong).

Reply:
Yes remove this.